setup.py

```diff
@@ -37,7 +37,7 @@
     ],
     "python_requires": ">=3.8,<3.12",
     "install_requires": requirements,
-    "extras_require": {"extra": ["graphix-ibmq", "graphix-perceval"]},
+    "extras_require": {'extra': ['graphix-ibmq', "graphix-perceval"], 'test': ['graphix-ibmq','scipy','random']},
```

Comment:
Please use " instead of ' if there is no reason!

Reply:
Don't know what you mean. If I remember correctly, random is no longer needed. I'll remove it.